lib/carma/client/client.rb

```diff
@@ -43,6 +38,34 @@ def create_submission_stub(_payload)
           }
         }
       end
+
+      def upload_attachments(payload)
+        with_monitoring do
+          client.post(
+            '/services/data/v47.0/composite/tree/ContentVersion',
+            payload,
+            'Content-Type': 'application/json'
+          ).body
+        end
+      end
+
+      def upload_attachments_stub(_payload)
```

Comment:
Can this stub be moved into the test suite? Not sure if there's a specific reason it needs to live here

Reply:
Ah, I see it's used with Flipper & stub_carma_responses currently

Reply:
Will add a comment here for clarity.